odk1-src/conf.py

```diff
@@ -160,6 +160,18 @@
     # Additional stuff for the LaTeX preamble.
     'preamble': '''
     \usepackage{fontspec}
+    \makeatletter
+      \\fancypagestyle{normal}{
+        \\fancyhf{}
+        \\fancyfoot[LE,RO]{{\py@HeaderFamily\\thepage}}
+        \\fancyhead[LO]{{\py@HeaderFamily\\nouppercase{\\rightmark}}}
+        \\fancyhead[RE]{{\py@HeaderFamily\\nouppercase{\\leftmark}}}
+        \\fancyfoot[CE,CO]{{\py@HeaderFamily{Get the latest guide at https://docs.opendatakit.org}}}
```

Comment:
It would be nice the text explained why and included a clickable link (if possible).

> We update our documentation daily, so this file is likely out of date! Get the most current version at [https://docs.opendatakit.org](https://docs.opendatakit.org).

Also, if possible, let's make the text not-bold and a little smaller since it's on every page.

Reply:
I agree with this.

Reply:
Sure, I will fix this!